.github/workflows/spread-tests.yaml

```
@@ -25,12 +25,42 @@ on:
         description: 'The rule .yaml file to use (found under tests/lib/spread/rules) for test discovery'
         required: true
         type: string
+      use-snapd-snap-from-master:
+        description: 'If true, will use the snapd snap built on the master branch'
+        required: false
+        type: boolean
+        default: false
+      spread-snapd-deb-from-repo:
+        description: 'If true, will use the snapd package from the repository when possible'
+        required: false
+        type: boolean
+        default: true
+      spread-experimental-features:
+        description: 'Experimental snapd features to enable'
+        required: false
+        type: string
 
 
 jobs:
   run-spread:
+    env:
+      SPREAD_SNAPD_DEB_FROM_REPO: ${{ inputs.spread-snapd-deb-from-repo }}
+      SPREAD_EXPERIMENTAL_FEATURES: ${{ inputs.spread-experimental-features }}
+      
     runs-on: ${{ fromJSON(inputs.runs-on) }}
     steps:
+
+    - name: Verify inputs
+      run: |
+          if [ "${{ inputs.backend }}" == "" ]; then
```

```suggestion
          if [ "${{ inputs.backend }}" = "" ]; then
```

Consider using -z as it is already used in the same file.